app/views/playlists/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="playlist">
+	<h1><%= @playlist.title %></h1>
+	<img src="<%= @picture %>"/>
+	<a href="/playlists/<%=@playlist.id%>/songs/new">Add a Song</a>
+	
+	<ul id="playlist">
+		<% @playlist.songs.each do |song| %>
+			<%= render partial: 'songs/mini_groove_song_card', locals: {title: song.title, artist: song.artist, url: @client.get_song_url_by_id(song.link), coverart: song.coverart}%>
```

Comment:
Use [Rails's built in collection](http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections) to render a partial for each item in a collection.